### PR TITLE
Group edit form

### PR DESCRIPTION
**Pull request by chdorner**
This adds a new group edit form, allowing group creators to update a group's name and description.

### CODE REVIEW

**Comment by chdorner** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;XCGrsTfv&#x2F;423-group-edit-form"> Group edit form</a>

File: h/templates/layouts/group.html.jinja2 [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{% extends "h:templates/layouts/base.html.jinja2" %}
+
+{% block page_title %}{{ page_title }}{% endblock %}
+
+{% block content %}
+  {{ panel('navbar') }}
+
+  <div class="content paper">
+    <div class="form">
```
Comment (robertknight) [inline h/templates/layouts/group.html.jinja2]: I suggest we get https://github.com/hypothesis/h/pull/3772 merged first and change the class name here to `form-container`

**Comment by robertknight** [comment]
As noted in Slack, we have a minor problem in that if you name a group `edit` you can't view its overview page because of the URL conflict.

**Comment by robertknight** [comment]
LGTM

File: h/templates/layouts/group.html.jinja2 (hunk 1 above)
Comment (robertknight) [inline h/templates/layouts/group.html.jinja2]: On second thoughts I'll just update #3772 after this is merged

**Pull request merged**